### DISCUSSION TITLE
Left-align diagnostic text

### DISCUSSION THREAD

**Discussion by helmesjo** [helix-editor/helix#10209] [Ideas]
When looking at diagnostics from the language server the text-box is displayed in the top right-hand side, but it's also right-aligned making it awkward to read:

![image](https://user-images.githubusercontent.com/2501070/223544543-a01ed8b2-e05c-4f2d-8789-54d467031e4f.png)

I'm not aware of any previous discussions regarding this (or if it's used for other things as well), but at least in situations as above it should just be left-aligned to be readable with anything but trivial diagnostics.

**Comment by kirawi** [comment]
There is https://github.com/helix-editor/helix/pull/6059

**Comment by helmesjo** [comment]
Ah, thanks! Though since PRs tend to lack useful descriptions, is that PR replacing the message output or just extending with some opt-in feature (eg. additional tools would be required besides a language-server or I'd have to enable it through in the config)? If not, then great, I can close this. Else I still think fixing up the message output would be useful. Not sure when right-aligned is ever useful for, well, text in general.

**Comment by kirawi** [comment]
It'll be replacing it I believe, and it doesn't requrie anything extra.

**Comment by helmesjo** [comment]
#6417 seems to be the one to wait for.
